### DISCUSSION TITLE
Linux?

### DISCUSSION THREAD

**Discussion by TheUnChosenOne** [stohrendorf/CroftEngine#286] [Q&A]
is there a way i can install on linux?

**Comment by stohrendorf** [comment]
If you're on openSUSE, you can use this: https://build.opensuse.org/package/show/games/croftengine - otherwise, you have to build it yourself.

**Reply by TheUnChosenOne** [reply]
i tryed building this program my self, i get an error,

CMake Error at cmake/get_utf8cpp.cmake:11 (message):
  utf8cpp not found
Call Stack (most recent call first):
  CMakeLists.txt:63 (include)

**Reply by stohrendorf** [reply]
You need to install `libutfcpp-dev`. The full list of dependencies for a debian-based system is `libboost-all-dev libglfw3-dev libpng++-dev libopenal-dev libglvnd-dev libfreetype6-dev cimg-dev ffmpeg libutfcpp-dev gettext libavcodec-dev libavformat-dev libavfilter-dev qt5-default qtbase5-dev qtbase5-dev-tools qttools5-dev libarchive-dev`.
